public/javascript/admin.js

```diff
@@ -359,7 +367,8 @@ function addMessage(userId, messageObject) {
                     chat.alert = false;
                     messageSide = 'right';
                 }
-                currentChat.innerHTML = currentChat.innerHTML + createMessageDiv(messageSide, messageObject.message);
+                $("#typingIcon").before(createMessageDiv(messageSide, messageObject.message));
```

Comment:
Does this do the same thing as the commented out code? Should the comment be deleted?